package.json

```diff
@@ -15,13 +15,13 @@
     "update-actions-build": "node actions/build.js"
   },
   "dependencies": {
-    "@actions/core": "^1.2.4",
+    "@actions/core": "^1.2.7",
     "@actions/github": "^2.2.0",
-    "axios": "^0.19.2",
-    "dotenv": "^6.0.0",
-    "google-auth-library": "^6.0.0",
-    "googleapis": "^51.0.0",
-    "newrelic": "^6.6.0",
+    "axios": "^0.21.1",
+    "dotenv": "^9.0.2",
+    "google-auth-library": "^7.0.4",
```

Comment:
I think you should manually test the CLA features.

Reply:
https://github.com/mridul-netizen/oppia-bot-test-repo-/pull/17/checks?check_run_id=2777219643 the tests are passing!

Reply:
https://github.com/mridul-netizen/oppia-bot-test-repo-/runs/2777213618?check_suite_focus=true in case the user has not signed the CLA the test fails!